code/obj/item/clothing/shoes.dm

```diff
@@ -446,6 +446,12 @@
 			. = "Looks like some big shoes to fill!"
 		. = ..()
 
+/obj/item/clothing/shoes/swat/superheavy // so heavy you can't get shoved!
+	name = "hi-grip assault boots"
+	desc = "Specialist combat boots designed to provide enhanced grip and ankle stability."
+	icon_state = "swatheavy"
+	magnetic = 1
```

Comment:
Magnetic only seems to impact push/pulling.  

Disarm/Shove appear to disregard this, deflection property can circumvent those but only up to 50% chance of ignoring it.